libcodechecker/server/api/report_server.py

```diff
@@ -1987,7 +1987,7 @@ def massStoreRun(self, name, tag, version, b64zip, force):
                     # second time here. Use re-use the
```

Comment:
Where is the first plist parse? Shouldn't that be updated too? Or the comment is outdated?

Reply:
The first plist parse is at the client side. We don't have to update those too.

Reply:
In that case this comment is outdated here right? We can not reuse the results from the client side.